### PR TITLE
Defocus found paths on dismissal of path activity process

### PR DESCRIPTION
**Pull request by joeybrk372**
- [ ] @sfeng88 @rygim @jharwig 
- [x] @srfarley @rajanpardipuramv5 @joeferner
- [x] @EvanOxfeld @joeybrk372

### CODE REVIEW

**Comment by sfeng88** [comment]
Does this work when the edges are not on the graph?

File: web/war/src/main/webapp/js/activity/activity.js [hunk 1]
```diff
@@ -184,6 +184,13 @@ define([
         this.removeTask = function(taskId) {
             this.removedTasks[taskId] = true;
             var task = this.tasksById[taskId];
+            var handler = handlersByType[task.type];
+
+            if (_.has(handler, 'remove')) {
+                var removeHandler = _.bind(handler.remove, this);
```
Comment (jharwig) [inline web/war/src/main/webapp/js/activity/activity.js]: just need to `handler.onRemove.call(this)`

File: web/war/src/main/webapp/js/activity/activity.js [hunk 2]
```diff
@@ -184,6 +184,13 @@ define([
         this.removeTask = function(taskId) {
             this.removedTasks[taskId] = true;
             var task = this.tasksById[taskId];
+            var handler = handlersByType[task.type];
+
+            if (_.has(handler, 'remove')) {
```
Comment (jharwig) [inline web/war/src/main/webapp/js/activity/activity.js]: change to `onRemove`

File: web/war/src/main/webapp/js/activity/handlers.js [hunk 3]
```diff
@@ -40,6 +40,9 @@ define([], function() {
                     });
                 });
             },
+            remove: function() {
```
Comment (jharwig) [inline web/war/src/main/webapp/js/activity/handlers.js]: Change to `onRemove`

**Comment by jharwig** [comment]
@sfeng88 Yes, because it's just a call to defocus. This is nicer in that it removes on dismiss instead of waiting for the user to click in the graph to remove (which sends the same event).

File: web/war/src/main/webapp/js/activity/activity.js (hunk 2 above)
Comment (EvanOxfeld) [inline web/war/src/main/webapp/js/activity/activity.js]: I tend to favor vanilla JS unless the library adds value in some way e.g. `if(handler.remove) {`.

Comment (jharwig) [inline web/war/src/main/webapp/js/activity/activity.js]: Yup, I prefer handler.remove over _.has, but _.isFunction provides some value here.

Comment (EvanOxfeld) [inline web/war/src/main/webapp/js/activity/activity.js]: @jharwig I didn't see your comment when I was writing  -- agreed, `_.isFunction` is a good way to go here as well. I recommend logging an error if `handler.onRemove` exists but is not a function.

**Pull request merged**